frontend/src/layout/navigation-3000/Navigation.scss

```diff
@@ -64,7 +71,7 @@
         }
 
         li + li {
-            margin-top: 1px;
+            margin-top: 0.25rem;
```

Comment:
This is quite a lot space, and actually with this new style I'd say the navbar would look awesome with _zero_ of this margin. When these elevated buttons sit right next to each other, they have this nice physicality to them because they can very slightly overlap